examples/test/videosphere/index.html

```diff
@@ -9,15 +9,41 @@
   <body>
     <a-scene stats>
       <a-assets>
-        <video id="lego" preload="true"
-               src="https://s3-us-west-2.amazonaws.com/s.cdpn.io/t-197/walking_the_dog_360_video_short.webm"
+        <!-- Test anonymous video with source element. -->
```

Comment:
`anonymous` -> `cross-origin (anonymous)` perhaps? even as someone familiar with browsers' cross-origin policies, this comment doesn't help me a whole lot. perhaps adding a link to MDN might be helpful?